server/src/api/EventRoutes.ts

```
@@ -207,4 +207,41 @@ eventRoutes.get<void, MyEventResponse>(
   }
 );
 
+eventRoutes.delete("/delete/:eventId", async (req, res) => {
+  const eventId = parseInt(req.params.eventId);
+  const userId = req.body.user.id;
+
+  try {
+    const event = await prisma.event.findUnique({
```

just delete and fail silently, this endpoint should be like 10 lines max.